cmd/untar.go

```diff
@@ -259,6 +260,11 @@ func untar(ctx context.Context, r io.Reader, putObject func(reader io.Reader, in
 			continue
 		}
 
+		// If zero or earlier modtime, set to current.
```

Comment:
Can you update the comment to say this is for snowball in particular ?

Reply:
Added the reason for the modification.